ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master-presubmits.yaml

```diff
@@ -254,6 +254,80 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )4.8-images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
Does this intentionally always run?

Reply:
nop, and there is a `name:` renaming that is at the wrong place, I'm fixing both of them